src/lazy/text/buffer.rs

```diff
@@ -30,20 +30,25 @@ use crate::{IonError, IonResult, IonType, TimestampPrecision};
 
 impl<'a> Debug for TextBufferView<'a> {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        const CHARS_TO_SHOW: usize = 64;
```

Comment:
🗺️ Changes in this method are just introducing a named constant and bumping the number of characters printed from `32` to `64`.